lib/dialects/mssql/query-generator.js

```diff
@@ -570,7 +570,14 @@ module.exports = (function() {
 
       if (options.limit || options.offset) {
         if (!options.order || (options.include && !subQueryOrder.length)) {
-          fragment += ' ORDER BY ' + this.quoteIdentifier(model.primaryKeyAttribute);
+          if(!isSubQuery && options.order) {
+            fragment += ', ';
```

Comment:
also on a completely aesthetic level you can save a bunch of lines rewriting this as:
```
fragment += (options.order && !isSubQuery) ? ', ' : ' ORDER BY ';
fragment += this.quoteIdentifier(model.primaryKeyAttribute);
```

Any saved lines helps a little bit in these huge files :smile: